File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int left = 0; 
        int right = numbers.size() - 1; 
        vector<int> Final; 

        while (left <= right){
            if (numbers[right] + numbers[left] < target){
                left++; 
            }
            else if (numbers[right] + numbers[left] > target){
                right--; 
            }
            else if (numbers[right] + numbers[left] == target){
                Final.push_back(left + 1); 
                Final.push_back(right + 1);
                break; 
            }
        }
        return Final; 
       
        
    }
};

File: 0167-two-sum-ii-input-array-is-sorted/0167-two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        arr = []
        left = 0
        right = len(numbers) - 1
        while left < right:
            Sum = numbers[left] + numbers[right]

            if Sum == target:
                arr += [left + 1]
                arr += [right + 1]
                return arr
            elif Sum < target:
                left += 1
            elif Sum > target: 
                right -= 1

        return arr


